src/database/models/user.js

```diff
@@ -5,7 +5,19 @@ export default (sequelize, DataTypes) => {
     email: DataTypes.STRING,
     username: DataTypes.STRING,
     password: DataTypes.STRING,
-    is_verified: DataTypes.BOOLEAN
+    is_verified: DataTypes.BOOLEAN,
+    role_value: DataTypes.INTEGER,
+    phone: DataTypes.STRING,
```

Comment:
isn't this supposed to be an int?

Reply:
this is for certain inputs thay sometimes looks like this `12-1234-12345`